src/shared/api.ts

```diff
@@ -46,14 +49,32 @@ export interface ModelInfo {
 	contextWindow?: number
 	supportsImages?: boolean
 	supportsComputerUse?: boolean
-	supportsPromptCache: boolean // this value is hardcoded for now
+	supportsPromptCache: boolean
 	inputPrice?: number
 	outputPrice?: number
 	cacheWritesPrice?: number
 	cacheReadsPrice?: number
 	description?: string
 }
 
+// DeepSeek
+export type DeepSeekModelId = keyof typeof deepSeekModels
+export const deepSeekDefaultModelId: DeepSeekModelId = "deepseek-chat"
+export const deepSeekModels = {
+	"deepseek-chat": {
+		maxTokens: 4096,
+		contextWindow: 64_000,
```

Comment:
isn't it 64k in their UI and 128k via their API?

Reply:
![CleanShot 2024-12-22 at 07 32 19@2x](https://github.com/user-attachments/assets/c2df2b7f-8100-4599-b820-44ebe35a0ecb)